src/localProxy.mjs

```diff
@@ -0,0 +1,20 @@
+import corsProxy from 'cors-anywhere'
+
+// Listen on a specific host via the HOST environment variable
+var host = process.env.HOST || '0.0.0.0';
```

Comment:
Can't this be a `const`?

Reply:
Yes, I used their template from their README, it is now adjusted in [commit aef1d05](https://github.com/SolidLabResearch/generic-data-viewer/pull/78/commits/aef1d0598e844c3baa550d380f89b397135d0980)